### PR TITLE
Nat gateway id not reflect in route table issue solved.

### PR DESCRIPTION
**Pull request by hardi171**
Intent of this PR, reflect NatGatewayId in the route table with other GateWayID. 

Problem was : the customer has one route with same destination cidr gateway=nat gateway and another with the same destination cidr but getway=vpn gateway, but the response we get from erlcloud when describing route tables doesn’t reflect.

### CODE REVIEW

File: src/erlcloud_ec2.erl [hunk 1]
```diff
@@ -1989,7 +1989,12 @@ extract_route(Node) ->
 extract_route_set(Node) ->
     [
      {destination_cidr_block, get_text("destinationCidrBlock", Node)},
-     {gateway_id, get_text("gatewayId", Node)},
+%%   {gateway_id, get_text("gatewayId", Node)},
```
Comment (kkuzmin) [inline src/erlcloud_ec2.erl:1992]: for the sake of simplicity I suggest we just have both
```
{gateway_id, get_text("gatewayId", Node)},
{nat_gateway_id, get_text("natGatewayId", Node)},
```

File: rebar.lock [hunk 2]
```diff
@@ -1,4 +1,4 @@
-{"1.1.0",
+{"1.2.0",
```
Comment (kkuzmin) [inline rebar.lock:1]: please revert.

**Review by kkuzmin — CHANGES_REQUESTED** [review]

File: rebar.lock (hunk 2 above)
Comment (hardi171) [reply rebar.lock:1]: ok

File: src/erlcloud_ec2.erl (hunk 1 above)
Comment (hardi171) [reply src/erlcloud_ec2.erl]: done.

**Review by motobob — APPROVED** [review]

**Review by kkuzmin — APPROVED** [review]

**Pull request merged**